charts/flyte-core/values.yaml

```diff
@@ -43,11 +43,6 @@ flyteadmin:
       memory: 50Mi
   # -- Default regex string for searching configuration files
   configPath: /etc/flyte/config/*.yaml
-  # -- Initial projects to create
```

Comment:
Won't this break backwards compatibility with old values files?

Reply:
If a user updates the HELM chart and the values set for the customer remain, it will not do any harm, because the projects were already created before. Projects created once are not deleted, if they do not show up on this list. 

So only if the user wants to create new projects, he or she needs to adapt the new values for the initial projects